lighthouse-core/scripts/lantern/minify-trace.js

```diff
@@ -81,6 +81,7 @@ function filterOutUnnecessaryTasksByNameAndDuration(events) {
   const {pid} = TracingProcessor.findMainFrameIds(events);
 
   return events.filter(evt => {
+    // @ts-ignore - TODO(cjamcl) #7790
```

Comment:
This is a little concerning. I don't have great confidence that we can solve the second order assertions here, at least not without a lot of machinery that makes a simple thing (current impl) a lot more difficult to follow. It's possible it'll be easy, though

Reply:
how's that?